packages/api-browser/window.js

```diff
@@ -3,9 +3,26 @@ if (!window.ssf) {
 }
 
 class Window {
-  constructor(...args) {
-    window.open(...args);
+  constructor(url, name, features) {
+    window.open(url, name, objectToFeaturesString(features));
   }
 }
 
+const objectToFeaturesString = (features) => {
+  let featuresString = '';
+
+  Object.keys(features).forEach((key, index) => {
+    // Need to convert booleans to yes/no
+    if (features[key] === true) {
+      features[key] = 'yes';
```

Comment:
I don't think that we should mutate parameters, as the side effects can lead to confusing behaviour. Maybe we could use `map`, instead of `forEach` to create an array of strings (one string for each feature), then use `arr.join(',')` to create the comma separate list as a string?